### PR TITLE
added tab functionality to separate info, updates/map, and data on pr…

### PR DESCRIPTION
**Pull request by malawi-mike**
…oject profiles but need help getting the content to render properly within tabs

### CODE REVIEW

File: project.php [hunk 1]
```diff
@@ -4,6 +4,11 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.0/css/materialize.min.css" rel="stylesheet" />
```
Comment (jdepree) [inline project.php]: don't we already have these in header.inc?

**Comment by malawi-mike** [comment]
you're right about the updates tab being too crowded.  would be great to divide updates page across two tabs -- one for the awesome scrolling update pics and the other for the map with a shadow circle over the corresponding village.  for this page, i'll continue working on adding parters and adding graphs for each development category.  i played with making the score graphs (not dollars invested) in the data tab show percentage change with 2014 as the baseline.  how hard would that be for you to implement.  i confirmed via testing that chart.js allows formulas in data fields (e.g., ((26-23)/23)*100.

**Comment by jdepree** [comment]
Splitting between updates and map makes sense.  Nested scrolling is pretty gross from a design and usability standpoint.  Making a graph shouldn't be very hard.

**Comment by jdepree** [comment]
It looks like, at least in Chrome, if you load the graphs in an initially non-visible tab, they never actually display.  Are you seeing this?

**Pull request merged**